test/msw-api/life-cycle-events/regression/null-body.test.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path'
+import { runBrowserWith } from '../../../support/runBrowserWith'
+import { captureConsole } from '../../../support/captureConsole'
+import { sleep } from '../../../support/utils'
+
+function createRuntime() {
+  return runBrowserWith(path.resolve(__dirname, 'null-body.mocks.ts'))
+}
+
+test('lifecycle mothod should support null body', async () => {
+  const runtime = await createRuntime()
+  const { messages } = captureConsole(runtime.page)
+
+  await runtime.request({
```

Comment:
And without the fix this test throws an exception upon trying to construct a `Response` instance internally?

Reply:
Yes right, If I add 

```js
runtime.page.on('pageerror', (err) => {
    console.log('pageerror: ', err)
  })
```

The console log is called with the error and no warning message is printed . I could add an additional check, what do you think?

Reply:
I think we can't go wrong with including this behavior into the assertions directly. If the error behavior can be observed by an exception thrown on the page, the test should assert that exception doesn't get thrown. Ideally, it doesn't even assert anything else.

Reply:
I have updated the assertion, removing the warning log